posthog/api/test/test_file_system.py

```diff
@@ -1,10 +1,56 @@
 from rest_framework import status
+from posthog.api.file_system import has_permissions_to_access_tree_view
 from posthog.test.base import APIBaseTest
 from posthog.models import FeatureFlag, Dashboard, Experiment, Insight, Notebook
-from posthog.models.file_system import FileSystem
+from posthog.models.file_system.file_system import FileSystem
+from unittest.mock import patch
+from rest_framework.exceptions import PermissionDenied
+
+
+class TestFileSystemAPIPermissions(APIBaseTest):
+    def setUp(self):
+        super().setUp()
+        self.user.is_staff = False
+        self.user.save()
+
+    @patch("posthoganalytics.feature_enabled", return_value=False)
```

Comment:
very nitpicking you could `parameterized.expand` these and it might be clearer but teeny thing